build/edge/docker-compose.yaml

```diff
@@ -18,22 +24,24 @@ services:
        dockerfile: ./build/edge/Dockerfile
        args: 
         - BUILD_FROM=${ARCH}/golang:1.12-alpine3.9
-        - RUN_FROM=${ARCH}/alpine:3.9
+        - RUN_FROM=${ARCH}/docker:dind 
         - QEMU_ARCH=${QEMU_ARCH}
     depends_on:
       - emqx
+      - dind
     environment:
       mqtt.server: tcp://emqx:1883
       edgehub.websocket.url: wss://${CLOUDHUB}/e632aba927ea4ac2b575ec1603d56f10/${EDGENAME}/events 
       edged.hostname-override: ${EDGENAME}
       edgehub.controller.node-id: ${EDGENAME}
       edgehub.websocket.certfile: ${CERTFILE} 
       edgehub.websocket.keyfile: ${KEYFILE}
+      edged.docker-address: tcp://dind:2375
+      DOCKER_HOST: tcp://dind:2375
```

Comment:
What's the difference between `edged.docker-address` and `DOCKER_HOST`? The configurations here are the same.

Reply:
`edged.docker-address` is the configuration of the modified `kubeedge/edge/conf/edge.yaml`. See [03-configmap-edgenodeconf.yaml](https://github.com/zhanghongtong/kubeedge/blob/75089f5b1cb5a13abcbef015285723ce47f7724d/build/edge/kubernetes/03-configmap-edgenodeconf.yaml#L44). `DOCKER_HOST` is to configure the docker server through environment variables. See [04-deployment-edgenode.yaml](https://github.com/zhanghongtong/kubeedge/blob/75089f5b1cb5a13abcbef015285723ce47f7724d/build/edge/kubernetes/04-deployment-edgenode.yaml#L35). This is all for docker-in-docker